infra/variables.tf

```diff
@@ -4,6 +4,13 @@ variable "region" {
   default     = "us-west-2"
 }
 
+variable "AWS-accountID" {
+  description = "The default AWS region to use for provisioning infrastructure"
```

Comment:
The description is incorrect, right? This is the account id. Should this be secret as well?

Reply:
cleaned up completed. Account# need not be in the secrets manager